templates/default/components/MDX.js

```diff
@@ -15,24 +15,25 @@ const components = {
 
 export default function MdxPage({ children, ...rest }) {
   const { Component, frontMatter } = children;
+  const seoImageUrl = siteConfig.authorUrl + frontMatter.image
```

Comment:
What happens if frontMatter.image is an absolute url and not relative?

Reply:
I will add the check for this 👍